build.teamcity.ps1

```diff
@@ -0,0 +1,72 @@
+# SPDX-License-Identifier: Apache-2.0
+# Licensed to the Ed-Fi Alliance under one or more agreements.
+# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+# See the LICENSE and NOTICES files in the project root for more information.
+
+$ErrorActionPreference = 'Stop'
+
+& "$PSScriptRoot/Initialize-PowershellForDevelopment.ps1"
+
+$teamcityParameters = Get-TeamCityParameters
+Write-Host
+Write-Host "$($teamcityParameters.Count) TeamCity parameters found."
+Write-Host
+
+# Build and Test
+$params = @{
+    InstallType   = Get-ValueOrDefault                    $teamcityParameters['odsapi.build.installType']    'Sandbox'
+    OdsTokens     = Get-ValueOrDefault (ConvertTo-Array   $teamcityParameters['odsapi.build.odsTokens'])     @()
+    Engine        = Get-ValueOrDefault                    $teamcityParameters['odsapi.build.engine']         'SQLServer'
+    NoCodeGen     = Get-ValueOrDefault (ConvertTo-Boolean $teamcityParameters['odsapi.build.noCodeGen'])     $false
+    NoRebuild     = Get-ValueOrDefault (ConvertTo-Boolean $teamcityParameters['odsapi.build.noRebuild'])     $false
+    NoDeploy      = Get-ValueOrDefault (ConvertTo-Boolean $teamcityParameters['odsapi.build.noDeploy'])      $false
+    RunPester     = Get-ValueOrDefault (ConvertTo-Boolean $teamcityParameters['odsapi.build.runPester'])     $true
+    RunDotnetTest = Get-ValueOrDefault (ConvertTo-Boolean $teamcityParameters['odsapi.build.runDotnetTest']) $true
+    RunPostman    = Get-ValueOrDefault (ConvertTo-Boolean $teamcityParameters['odsapi.build.runPostman'])    $true
+    RunSmokeTest  = Get-ValueOrDefault (ConvertTo-Boolean $teamcityParameters['odsapi.build.runSmokeTest'])  $true
+    UsePlugins    = Get-ValueOrDefault (ConvertTo-Boolean $teamcityParameters['odsapi.build.usePlugins'])    $false
+}
+
+Invoke-Task "Remove-EdFiDatabases" { Remove-EdFiDatabases -Force -Engine $params.Engine }
+
+Write-FlatHashtable $params
+Initialize-DevelopmentEnvironment @params
+
+# Package
+$params = @{
+    ProjectPath     = (Get-RepositoryResolvedPath (Get-ProjectTypes).SandboxAdmin)
+    PackageId       = Get-ValueOrDefault                  $teamcityParameters['odsapi.build.package.sandboxAdmin.id']
+    Version         = Get-ValueOrDefault                  $teamcityParameters['odsapi.build.package.sandboxAdmin.version']     '0.0.0'
+    Properties      = Get-ValueOrDefault (ConvertTo-Array $teamcityParameters['odsapi.build.package.sandboxAdmin.properties']) (Get-DefaultNuGetProperties)
+    OutputDirectory = Get-ValueOrDefault                  $teamcityParameters['odsapi.build.package.sandboxAdmin.output']      "$PSScriptRoot/packages"
+}
+New-WebPackage @params
```

Comment:
This can be a separate follow up ticket, but packaging should be part of initdev and just not run by default, similar to tests.  This is a lot of logic and powershell someone would have to duplicate in order to make another build system function.

Reply:
one of the reasons I put it here was so that all the TeamCity parameters would be exposed in this one file. I think it makes sense to move these if we go with a config object.